sktime/forecasting/compose/_reduce.py

```diff
@@ -43,23 +45,32 @@ def __init__(self, regressor, window_length=10, step_length=1):
         self.step_length_ = None
         self._cv = None
 
-    def update(self, y, X=None, update_params=False):
+    def update(self, y, X=None, update_params=True):
```

Comment:
Thinking about this, can we not simply remove this here as we inherit `update` from the base forecaster? I probably overrode it to handle the error when X is passed, but this should already be handled in `fit` and `update` only works on fitted forecasters, so we should be able to simply remove it here, no?